src/App.tsx

```diff
@@ -21,11 +52,14 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={handleSelect}
+            defaultValue={perPage}
           >
-            <option value="3">3</option>
-            <option value="5">5</option>
-            <option value="10">10</option>
-            <option value="20">20</option>
+            {[3, 5, 10, 20].map(option => (
```

Comment:
Better to move to separate variable
```suggestion
            {countOptions.map(option => (
```

Reply:
fixed ✅